### PR TITLE
docs: Remove euca reference

### PR DESCRIPTION
**Pull request by TheRealFalcon**
## Proposed Commit Message

```
docs: Remov euca reference and example script

We don't recommend Eucalyptus or include it any of our tutorials or
other examples. Including it without any context or explanation makes
no sense.
```

### CODE REVIEW

File: doc/rtd/explanation/format.rst [hunk 1]
```diff
@@ -50,22 +50,6 @@ archive.
 User data scripts can optionally render cloud instance metadata variables using
 jinja templating. See :ref:`instance_metadata` for more information.
 
-Example script
```
Comment (holmanb) [inline doc/rtd/explanation/format.rst:53]: Isn't this still a valid user data script? Why should we remove that part?

Comment (TheRealFalcon) [reply doc/rtd/explanation/format.rst:53]: Is there value to showing an arbitrary shell script?

Comment (holmanb) [reply doc/rtd/explanation/format.rst:53]: Not a ton, but I think there is some value. Without an example for each type, our format page is a dense wall of text that is harder to skim quickly.

I'd personally love to see each of our documented "User data format" types show - or link to in the case of gzip (if gzip is even a user data format) - a very simple example of a working user data of that format which could be passed to cloud-init, with an explanation of what it does.

Note: this section is still missing important information (do user-data scripts run on every boot? first boot? etc...)

Comment (holmanb) [reply doc/rtd/explanation/format.rst:53]: If you still want to remove it, I'm fine with that. It really doesn't add much on its own. That said, I'll probably add an example back in when implementing [this docs change](https://github.com/canonical/cloud-init/issues/4739) at some point in the future.

**Review by holmanb — APPROVED** [review]
Up to you. I'm fine with the change as-is, but would like to see eventually an example per format.

**Comment by TheRealFalcon** [comment]
I'll close this PR for now. It was just one of those things I saw when trying to find something else so figured I'd change it right away. Instead I'll add a comment to https://github.com/canonical/cloud-init/issues/4739

**Comment by holmanb** [comment]
> I'll close this PR for now. It was just one of those things I saw when trying to find something else so figured I'd change it right away. Instead I'll add a comment to #4739

Gotcha, sounds good +1

**Pull request closed without merging**